src/test/java/step_definitions/SalesForceCPQ.java

```diff
@@ -56,7 +56,7 @@ public void navigate_to_the_url() throws Exception {
     }
 
 
-    @When("^Navigate to Salesforce as \"([^\"]*)\"$")
+    @When("^navigate to Salesforce as \"([^\"]*)\"$")
```

Comment:
Please start the step definition from capital letter as in feature file all other steps are starting with capital letter, please do it for login as well